### PR TITLE
Added dynamic Prims and Kruskals algorithm.

### PR DESCRIPTION
**Pull request by Tanmay2510**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	<classpathentry kind="src" path="Java/Greedy_Algorithms"/>
+	<classpathentry excluding="countDigits/" kind="src" path="Java/math"/>
+	<classpathentry kind="src" path="Java/math/countDigits"/>
+	<classpathentry kind="src" path="Java/Operating-System"/>
+	<classpathentry kind="src" path="Java/other"/>
+	<classpathentry kind="src" path="Java/search"/>
+	<classpathentry kind="src" path="Java/sort"/>
+	<classpathentry kind="src" path="Miscellaneous/HackerRank"/>
+	<classpathentry kind="output" path="bin"/>
+</classpath>
```
Comment (HarshCasper) [inline .classpath]: Remove this

File: .gitignore [hunk 2]
```diff
@@ -154,3 +154,4 @@ venv.bak/
 dmypy.json
 .pyre/
 .pytype/
+/bin/
```
Comment (HarshCasper) [inline .gitignore]: Remove this

File: .project [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+	</natures>
+</projectDescription>
```
Comment (HarshCasper) [inline .project]: Remove this

File: Java/Algorithms/Kruskals_Algorithm.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+//import java.util.Scanner;
```
Comment (HarshCasper) [inline Java/Algorithms/Kruskals_Algorithm.java]: Add a beginner-friendly pitch about the Algorithm and what it does.

File: Java/Algorithms/Kruskals_Algorithm.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+import java.util.Arrays;
+import java.util.Scanner;
+class Edge implements Comparable<Edge>{
+	int source;
+	int destination;
+	int weight;
+	
+	//@Override
+	public int compareTo(Edge o) {
+		return this.weight - o.weight;  //To Sort in increasing order
```
Comment (HarshCasper) [inline Java/Algorithms/Kruskals_Algorithm.java]: Use comments above the line of code and not beside it

File: Java/Algorithms/Kruskals_Algorithm.java [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+	}
+}
+
+public class Kruskals_Algorithm {
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+	
+
```
Comment (HarshCasper) [inline Java/Algorithms/Kruskals_Algorithm.java]: Remove these extra lines of code

File: Java/Algorithms/Kruskals_Algorithm.java [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+public class Kruskals_Algorithm {
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+	
+	
+		int V = scan.nextInt();
+		int E = scan.nextInt();
+        Edge input[] = new Edge[E];
```
Comment (HarshCasper) [inline Java/Algorithms/Kruskals_Algorithm.java]: Improper indentation. Fix it!

File: Java/Algorithms/Kruskals_Algorithm.java [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+		
+			if(sourceParent != destParent) {
+				output[count] = currentEdge;
+				count++;
+				parent[sourceParent] = destParent;
+			}
+			i++;
+														
+		}
+		for(int j=0;j<v-1;j++) {
```
Comment (HarshCasper) [inline Java/Algorithms/Kruskals_Algorithm.java:53]: Poorly formatted Code. Use a Code formatter to format your Code properly

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Please make these changes initially.

**Comment by Tanmay2510** [comment]
Hey, Now can you review my pull request @HarshCasper

**Comment by HarshCasper** [comment]
Why is the `.gitignore` deleted? 

And make the README updates for all of your Code files.

**Comment by Tanmay2510** [comment]
There is no Readme file in Java algorithm folder @HarshCasper.

**Comment by Tanmay2510** [comment]
I have updated the Readme file can you check it @HarshCasper ?

**Comment by Tanmay2510** [comment]
> Why is the `.gitignore` deleted?
> 
> And make the README updates for all of your Code files.

Can you please help me with the .gitignore file ?

**Comment by HarshCasper** [comment]
> > Why is the `.gitignore` deleted?
> > And make the README updates for all of your Code files.
> 
> Can you please help me with the .gitignore file ?

What help is required?

**Comment by HarshCasper** [comment]
> I have updated the Readme file can you check it @HarshCasper ?

Nope. You have not updated it.

**Comment by Tanmay2510** [comment]
Are you able to see the changes @HarshCasper @atarax665 ?
Screenshot 2021-02-25 151147
Screenshot 2021-02-25 151147

**Comment by HarshCasper** [comment]
We are not familiar with this UI tool which you are using to visualize the Git branching and commits. Have a look [here](https://github.com/TesseractCoding/NeoAlgo/pull/2145/files) and see all the changes being pushed by you in your commits.

**Pull request closed without merging**